zoom_video_composer.py

```diff
@@ -130,6 +131,14 @@
     help="Resampling technique to use when resizing images.",
     show_default=True,
 )
+@click.option(
+    "-ss",
+    "--supersample",
```

Comment:
This needs to be `--super-sampling-factor` to work.

Reply:
Fixed